crates/matrix-sdk/src/event_cache/pagination.rs

```diff
@@ -259,6 +317,16 @@ impl RoomPagination {
     }
 }
 
+/// A type representing whether the timeline has been reset.
+#[derive(Debug)]
+pub enum TimelineHasBeenResetWhilePaginating {
```

Comment:
Roses are red,
Violets are blue,
Unnamed booleans suck,
All hail enums.

:heart_hands: 

The name is a bit of a mouthful though, can you think of a shorter one please?

Reply:
I thought about `TimelineHasBeenReset` but I'm afraid it can create confusion if used somewhere else. Let's keep the name as is, and let's refactor if needed.